ui/src/App.vue

```diff
@@ -50,6 +50,8 @@ export default {
     logOut() {
       Cookies.remove("sh_authtoken");
       Cookies.remove("sh_user");
+      this.$store.state.token=undefined;
```

Comment:
Vuex state shouldn't be changed directly, but using [mutations](https://vuex.vuejs.org/guide/mutations.html). You can find the mutations to set the user and token in [ui/src/store/index.js](https://github.com/chaoss/grimoirelab-sortinghat/blob/3abf716759c0af6aa9f2f14cd7f9bb82578e0623/ui/src/store/index.js)

Reply:
Ok, I will do that.

Reply:
Done 👍🏻